2019/12/27/from-dev-to-pm.html.md

```diff
@@ -0,0 +1,233 @@
+If your client is very opinionated about their project, try to work out directions and methods
+BEFORE allowing comms between devs and client. (Use Trello, Slack, etc to assign specific tasks for
+specific people) Try to track progress of predefined tasks rather than agile development.
+
+If your client is tech savvy, easy to get along with, and you can convince your devs of that,
+direct interaction can be extremely productive. But don't just assume this will work, be on the
+the first phone calls or Zoom meetings.. observe the dynamics... try to be a fly on the wall..
+don't run the meeting, step in only if necessary.
+
+Use your resources. If you have 2 front end people, and you have friction between the client and
+one of them, introduce the other for a separate task. Observe, did it work out? Do I need to
+phase out one in favor of another?
+
+If possible never "pull" a developer completely off a project, unless of course it's requested /
+required. (do the words "burning your bridges" sound familiar?)
+
+Always talk to the developer in private about whatever the issue is, and always point out to
+developers that relationships do not depend on talent alone. What works for one client will
+not work for the next.
+
+Make sure you know your developers. You need to stand up for them when talking to clients. If
+you are not comfortable with a developer, you need to either discuss it with a superior, or if
+you are the superior you need to let them go. You cannot lie to a client about your faith in
+a developer. Clients don't want to hear you bad mouth a developer in your own company ... this
+simply makes you and the developer look bad. The client expects you to have faith in your dev,
+and to stand up for them.
+
+With that said, back to rule #1, know your client. You have to know what that client will
+tolerate or expect. You may have to be firm with some clients, so that you can give them what
+is the best solution for their project.
+
+Other clients may require a soft approach and may completely understand "how this stuff works".
+When you get one of those, you will understand very quickly, and be very thankful.
+</pre>
+### Summary
+<pre>
+Managing projects is a challenging proposition. But if you put things in perspective,
+you will find that it can be simplified and rewarding. Try to keep the big picture in 
+mind as you proceed through the project. It's easy to get drawn into the details of a 
+project, and sometimes that is what is required. However, if you have good resources 
+available to you, it is up to you to put the areas of responsibility into the proper hands.
+Coming from developement, this will be a real temptation. But you have a new title now,
+so put it to work!
+
+Remember, you are not working on this project to create something that you like, or that 
+your developers like. You are creating something for your client. The projects goals
```

Comment:
not *only* that you like ... but I think the project manager and the developers should usually like what they're creating, at least within the sphere of the business need. If they hate it, that indicates a problem, even if the client likes it. It may mean other users will hate it too, which doesn't serve the client in the long run, so at least a conversation is in order.

Reply:
Reworded.